specs/features/Tables.md

```diff
@@ -0,0 +1,259 @@
+| --- | --- | --- | --- |
+| `horizontalCellContentAlignment` | `HorizontalAlignment` | No | Specifies how the content of each cell should be horizontally aligned by default within the column. Defaults to `left`. This value override the table setting, and can itself be overridden by setting `horizontalCellContentAlignment` on a row or by setting the `horizontalAlignment` property on each individual element in a cell. |
+| `verticalCellContentAlignment` | `VerticalAlignment` | No | Specifies how the content of each cell should be vertically aligned by default within the column. Defaults to `top`. This value overrides the table setting and can itself be overridden by setting `verticalCellContentAlignment` on a row or cell. |
+| `width` | `number` or `string` | No | Defines the width of the column. When expressed as a string, width represents an explicit number of pixels, and must be in the format "<positive number>px", e.g. "80px". When expressed as a number, width represents the weight of the column within the table. |
+
+#### Computing effective column widths
+- First, all explicitly sized columns are get the space they request
+- The remaining space is then split between weighted columns according to their individual weight
+  - Weights can be expressed as any positive number
+  - The sum of all weights in a Table does not have to equal 100
+
+### TableRow
+| Name | Type | Required | Description |
+| --- | --- | --- | --- |
+| `cells` | `TableCell[]` | No | Defines the cells for this particular row. See `TableCell` below. When a row doesn't contain as many cells as there are columns defined on the Table, the remaining columns remain empty for that row. If there are more cells in a row than there are columns defined on the Table, the extra cells are not rendered. |
+| `horizontalCellContentAlignment` | `HorizontalAlignment` | No | Specifies how the content of each cell should be horizontally aligned by default within the row. Defaults to `left`. This value override the table and column settings, and can itself be overridden by setting the `horizontalAlignment` property on each individual element in a cell. |
+| `verticalCellContentAlignment` | `VerticalAlignment` | No | Specifies how the content of each cell should be vertically aligned by default within the row. Defaults to `top`. This value overrides the table and column settings and can itself be overridden by setting `verticalCellContentAlignment` on a cell. |
+| `style` | `string` | No | Specifies the style of that row in terms of background color. `style` must be set to a valid container style name. When not specified, style is inherited from the row's parent container. |
+
+### TableCell
+`TableCell` extends `Container`, and thus inherits `Container`'s properties. It doesn't introduce additional properties.
+
+As a descendent of `Container`, `TableCell` can contain any Adaptive Card element, including ColumnSet or even other Table elements.
+
+### HostConfig changes
```

Comment:
this section doesn't mention the `cellSpacing` changes from the `Notes on rendering Table` above

Reply:
Looks like it was at the top. Moved it down, and added the json.